recipes/bettercam/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "bettercam" %}
+{% set version = "1.0.0" %}
+{% set python_min = "3.8" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default. 

```suggestion
```